wcosa/boards.json

```diff
@@ -1,100 +1,120 @@
 {
     "diecimila": {
+        "name": "Cosa Arduino Diecimila (ATmega168/BOOT_168)",
```

Comment:
Just realized now but you can read these directly from `cosa/boards.txt`, is there a reason we are keeping like copies of the config in JSON format?

Reply:
yeah, we can but that file is a bit hard to parse. So I have a parser which gathers essential information from that file and creates this file. Now we can just use this file. Also I don't have to parse the whole file whenever someone asks for a board

Reply:
Fair enough,